assets/js/script.js

```diff
@@ -0,0 +1,164 @@
+const myAPIKey = "f23ee9deb4e1a7450f3157c44ed020e1";
+const currentWeather = $("#current-weather");
+const searchContainer = $("#search-history");
+const forecastContainer = $("#forecast-container");
+
+const constructSearchHistory = function (array) {
+  array.forEach((element) => {
+    historyButton = `<li><button id="search-btn">${element}</button></li>`;
+    const searchHist = $("#search-history");
```

Comment:
Instead of declaring the variable first here just to be used on the line below, you can use the jQuery selector directly. `$("#search-history").append(historyButton)`

Reply:
I think I am able to do some better refactoring on this now